Gofer.NET.Utils/ActionExtensionMethods.cs

```diff
@@ -11,50 +12,50 @@ public static TaskInfo ToTaskInfo(this Expression<Action> expression)
         {
             var methodCallArgumentResolutionVisitor = new MethodCallArgumentResolutionVisitor();
             var expressionWithArgumentsResolved =
-                (Expression<Action>) methodCallArgumentResolutionVisitor.Visit(expression);
+                (Expression<Action>)methodCallArgumentResolutionVisitor.Visit(expression);
 
-            var method = ((MethodCallExpression) expressionWithArgumentsResolved.Body);
+            var method = ((MethodCallExpression)expressionWithArgumentsResolved.Body);
             var m = method.Method;
             var args = method.Arguments
                 .Select(a =>
                 {
-                    var value = ((ConstantExpression) a).Value;
-                    return value;
+                    var value = ((ConstantExpression)a).Value;
+                    return value is CancellationToken ? null : value;
```

Comment:
This type of special case to exclude any cancellation token from being serialized should not be necessary to enable the behavior for the task client properly accept a cancellation token.

Reply:
Also these types of special cases regarding the serialized function body+ arguments in general are not desirable.

Reply:
Agreed, I think it was a bad idea now. Removed:)